get-subseqs

```diff
@@ -1,3 +1,74 @@
-#!/bin/bash
+#!/usr/bin/python3
 
-python3 src/get-subseqs.py $@
+import sys
+import numpy as np
+
+def get_subseqs():
+
+    try:
+        inFile_1 = sys.argv[1]
+        inFile_2 = sys.argv[2]
```

Comment:
Instead of relying on exceptions, which can be hard to read, and duplicated code, which is a deadly sin, you should just check how many arguments you have to determine if `inFile_2` should be `sys.stdin`:

```python
f1_lines = open(sys.argv[1]).readlines()
f2_lines = (sys.stdin if len(sys.argv) < 3 else open(sys.argv[2])).readlines()
```

The `with` context manager is great and all, but the garbage collector should work adequately when you just want to read in all the lines from a file...